src/core/scene/a-scene.js

```diff
@@ -520,6 +520,10 @@ module.exports.AScene = registerElement('a-scene', {
           rendererConfig.antialias = this.getAttribute('antialias') === 'true';
         }
 
+        if (this.hasAttribute('logarithmicDepthBuffer')) {
```

Comment:
Let's initialize `logarithmicDepthBuffer` to false here: https://github.com/aframevr/aframe/pull/3717/files#diff-eb9bb1084863a2f65678ea67d6e7cf5eR518

Reply:
The logarithmicDepthBuffer option has been set to false by default!